src/ape/api/accounts.py

```diff
@@ -118,6 +119,9 @@ def call(self, txn: TransactionAPI, send_everything: bool = False) -> ReceiptAPI
         # else: Assume user specified the correct amount or txn will fail and waste gas
 
         if send_everything:
+            if txn.max_fee is None:
+                raise TransactionError(message="Max fee must not be null.")
```

Comment:
```suggestion
                raise TransactionError(message="Max fee must not be None.")
```

None is more pythonic?